solitude/urls.py

```diff
@@ -37,13 +19,15 @@
 )
 
 urls = [
+    url(r'^$', 'solitude.views.home', name='home'),
+    url(r'^generic/', include('lib.buyers.urls', namespace='buyer')),
```

Comment:
`buyer:buyer`, `seller:seller`, `transaction:transaction` are kinda naff. Might go for something a little easier like `generic:buyer`